docs/reference/commandline/build.md

```diff
@@ -300,3 +301,11 @@ like `ENV` values do.
 
 For detailed information on using `ARG` and `ENV` instructions, see the
 [Dockerfile reference](../builder.md).
+
+### Specify isolation technology for container (--isolation)
```

Comment:
@jhowardmsft I'll open a new issue and leave the code to you.
So this PR will be a pure docs PR.
I'll do it now.